src/Main.java

```diff
@@ -1,5 +1,81 @@
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+        System.out.println("home 3");
+
+        int age = 15;
+        if (age >= 18) {
+            System.out.println("Если возраст равен " + age + ", то Вы совершеннолетний");
+        }
+        if (age < 18) {
+            System.out.println("Если возраст равен " + age + ", Вы не достигли совершеннолетия, нужно немного подождать");
+        }
+        //задача 1 окончена
+
+        int temperature = 4;
+        if (temperature > 5) {
+            System.out.println("На улице " + temperature + " градуса" + ", нужно надеть шапку");
+        }
+        if (temperature < 5) {
+            System.out.println("На улице " + temperature + " градуса" + ", можно надеть шапку");
+        }
+        //задача 2 окончена
+
+        int speed = 40;
+        if (speed > 60) {
+            System.out.println("Если скорость выше " + speed + ", то придётся заплатить штраф");
+        }
+        if (speed < 60) {
+            System.out.println("Если скорость ниже " + speed + ", то можно ездить спокойно");
+        }
+        //задача 3 окончена
+
+        int age2 = 21;
```

Comment:
можно было не создавать новую переменную, а просто переиспользовать уже созданную
```suggestion
        age = 21;
```